projects/components/src/menu-dropdown/menu-dropdown.component.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  EventEmitter,
+  Input,
+  Output,
+  QueryList
+} from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { Observable } from 'rxjs';
+import { IconSize } from '../icon/icon-size';
+import { MenuItemComponent } from './menu-item.component';
+
+@Component({
+  selector: 'ht-menu-dropdown',
+  styleUrls: ['./menu-dropdown.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="menu-dropdown">
+      <ht-popover [closeOnClick]="true" class="menu-container">
+        <ht-popover-trigger>
+          <div class="trigger-content">
+            <ht-label *ngIf="this.label" class="trigger-label" [label]="this.label"> </ht-label>
+            <ht-icon *ngIf="this.icon" class="trigger-icon" [icon]="this.icon" size="${IconSize.Small}"></ht-icon>
+          </div>
+        </ht-popover-trigger>
+        <ht-popover-content>
+          <div class="dropdown-content">
+            <div *ngFor="let item of items" (click)="this.onMenuItemClick(item)" class="menu-item">
```

Comment:
What we discussed yesterday was instead of explicitly rendering these in the dropdown, move the icon into the item template and here just render the content via `ng-content`. This both makes the DOM more similar to the template but also removes the need for an output at all, since click can be directly bound by the user of menu-dropdown. See toggle button group for an example.

Reply:
@aaron-steinfeld I'm working on that. Ignore the last few commits. I had made these changes earlier. Just wanted to pull it in another project.